sample_workloads/lit-gpt-demo/helm/templates/litgpt.yaml

```diff
@@ -1,9 +1,10 @@
 {{- $requiredVar := .Values.cluster.nNodes | required ".Values.cluster.nNodes is required" -}}
 {{- $requiredVar := .Values.cluster.nodePool | required ".Values.cluster.nodePool is required" -}}
 {{- $requiredVar := .Values.network.ncclIfnames | required ".Values.ncclIfnames is required" -}}
-{{- $requiredVar := .Values.workload.gcsBucket | required ".Values.gcsBucket is required" -}}
 {{- $requiredVar := .Values.workload.jobTimestamp | required ".Values.jobTimestamp is required" -}}
+{{- $requiredVar := .Values.workload.gcsExperimentBucket | required ".Values.gcsExperimentBucket is required" -}}
```

Comment:
Thoughts on calling this LogBucket instead of ExperimentBucket? AFAIK we're only dumping logs in here.

Reply:
This is true, with the exception of `metrics.csv` with includes performance metrics. I think ExperimentBucket is more fitting because of this.